nhrpd/nhrp_peer.c

```diff
@@ -442,9 +586,9 @@ static void nhrp_handle_resolution_req(struct nhrp_packet_parser *pp)
 			       &pp->src_proto, &pp->dst_proto);
 
 	/* Copied information from request */
-	hdr->flags = pp->hdr->flags
-		     & htons(NHRP_FLAG_RESOLUTION_SOURCE_IS_ROUTER
-			     | NHRP_FLAG_RESOLUTION_SOURCE_STABLE);
+	hdr->flags =
+		pp->hdr->flags & htons(NHRP_FLAG_RESOLUTION_SOURCE_IS_ROUTER
+				       | NHRP_FLAG_RESOLUTION_SOURCE_STABLE);
```

Comment:
Whitespace change

Reply:
I ran clang-format over the code to fix up a bunch of bad formatting we had introduced. I will revert the places where it changed existing code only by whitespace.

Reply:
Makes sense, would've been fine then, although I can't say I'm a big fan  of how clang-format is changing this.